vector/src/main/java/im/vector/app/features/home/room/detail/timeline/item/MessageLiveLocationItem.kt

```diff
@@ -26,8 +26,8 @@ import im.vector.app.core.resources.toTimestamp
 import im.vector.app.core.utils.DimensionConverter
 import im.vector.app.features.home.room.detail.RoomDetailAction
 import im.vector.app.features.home.room.detail.timeline.style.TimelineMessageLayout
-import im.vector.app.features.location.live.LocationLiveMessageBannerView
 import im.vector.app.features.location.live.LocationLiveMessageBannerViewState
+import im.vector.app.features.location.live.LocationLiveRunningBannerView
 import org.threeten.bp.LocalDateTime
 
 @EpoxyModelClass
```

Comment:
Not a part of this PR but to be consistent, the class name should be `MessageLocationLiveItem` wdyt?

Reply:
I think I will create a dedicated PR to align the naming of all components. I think I will keep the order `LiveLocation`.